addon/lib/metrics.js

```diff
@@ -160,34 +137,33 @@ const Metrics = module.exports = {
     const dataParsed = variantMaker.parseTests(JSON.parse(data));
 
     store.experimentVariants[subject] = dataParsed;
-    Metrics.experimentFeaturesChanged(subject, dataParsed);
     Events.emit(EVENT_SEND_VARIANTS, {
       data: JSON.stringify(dataParsed),
       subject: self.id
     });
   },
 
   onExperimentPing: function(ev) {
+    const timestamp = Date.now();
     const { subject, data } = ev;
     const dataParsed = JSON.parse(data);
 
-    // TODO: Map add-on ID (subject) to other pingTypes as necessary
-    const pingType = 'testpilottest';
-
-    if (store.experimentVariants && subject in store.experimentVariants) {
-      dataParsed.variants = store.experimentVariants[subject];
-    }
-
-    const payload = {
-      version: 1,
-      test: subject,
-      payload: dataParsed
-    };
+    AddonManager.getAddonByID(subject, addon => {
```

Comment:
@bwinton - I was talking to @ilanasegall the other day and she was not a fan of our batching in tab center either.  She says she's working around it on the server side, but if we can change it, I imagine it'd be good.  If she'd rather not mess with it, we could skip it also and just not have a version for tab center.  

@chuckharmston - so, worst case scenario if someone implements the metrics poorly is that version is blank?  The pings still happen other than that?  If that's the case, I think we're fine with this.  We're working with developers so, just like any other API of ours, if they mis-code it we can adjust the experiment.  We should write that in the docs somewhere though.